plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/IcebergGlueCatalogModule.java

```diff
@@ -60,7 +60,7 @@ protected void setup(Binder binder)
         install(conditionalModule(
                 IcebergGlueCatalogConfig.class,
                 IcebergGlueCatalogConfig::isSkipArchive,
-                config -> requestHandlers.addBinding().toInstance(new SkipArchiveRequestHandler())));
+                internalBinder -> requestHandlers.addBinding().toInstance(new SkipArchiveRequestHandler())));
```

Comment:
rename is good, but please also change `requestHandlers.addBinding` to `newSetBinder(internalBinder,..).addBinding`